<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>791</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_transfer">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>转账</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>89</y>
      <width>400</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>金额：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_amount"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>手续费：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fee"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>AITD</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>721</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>源地址：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_src_address"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>源私钥：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_src_privkey"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>目标地址：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dst_address"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>目标私钥：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dst_privkey"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionaccount"/>
    <addaction name="actiontoken"/>
    <addaction name="actionbridge"/>
    <addaction name="actionairdrop"/>
    <addaction name="actionquantization"/>
    <addaction name="actionborrow_lend"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actionasset"/>
    <addaction name="actionrecord"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionrpc"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actiontoken">
   <property name="text">
    <string>代币</string>
   </property>
  </action>
  <action name="actionbridge">
   <property name="text">
    <string>桥</string>
   </property>
  </action>
  <action name="actionaccount">
   <property name="text">
    <string>账户</string>
   </property>
  </action>
  <action name="actionairdrop">
   <property name="text">
    <string>空投</string>
   </property>
  </action>
  <action name="actionquantization">
   <property name="text">
    <string>量化</string>
   </property>
  </action>
  <action name="actionasset">
   <property name="text">
    <string>资产</string>
   </property>
  </action>
  <action name="actionrecord">
   <property name="text">
    <string>记录</string>
   </property>
  </action>
  <action name="actionrpc">
   <property name="text">
    <string>节点</string>
   </property>
  </action>
  <action name="actionborrow_lend">
   <property name="text">
    <string>借贷</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
